src/Provider/AbstractProvider.php

```diff
@@ -442,36 +471,25 @@ public function getAccessToken($grant, array $options = [])
 
 
     /**
-     * Get a request instance.
+     * Get a request instance, optionally authenticated using an access token.
      *
      * Creates a PSR-7 compatible request instance that can be modified.
-     * The request is not automatically authenticated.
      *
      * @param  string $method
      * @param  string $url
+     * @param  AccessToken|string $token
      * @param  array  $options Any of "headers", "body", and "protocolVersion".
      * @return RequestInterface
      */
-    public function getRequest($method, $url, array $options = [])
+    public function getRequest($method, $url, $token = null, array $options = [])
```

Comment:
Having thought about this for a while, I agree with you. 

See bbfe83e3d01f8af39e1d7f379b4d0ace2f55de3d